website/content/en/preview/tasks/pod-density.md

```diff
@@ -0,0 +1,59 @@
+---
+title: "Control Pod Density"
+linkTitle: "Control Pod Density"
+weight: 20
+---
+
+Pod density is the number of pods per node. 
+
+Kubernetes has a default limit of 110 pods per node. Karpener shares this default. 
+
+## Max Pods
+
+Do not use the `max-pods` argument to kubelet. Karpenter is not aware of this value. For example, Karpenter may provision an instance expecting it to accommodate more pods than this static limit. 
+
+## Increase Pod Density
+
+### Networking Limitations 
+
+*☁️ AWS Specific*
+
+The number of pods on a node is limited by the number of networking interfaces (ENIs) that may be attached to a node. 
+
+AWS VPC CNI v1.9 introduced prefix assignment. In short, a single ENI can provide IP addresses for multiple pods. Much higher pod densities are now supported. 
+
+Run the Karpenter controller with the enviornment variable `AWS_ENI_LIMITED_POD_DENSITY` (or the argument  `--aws-eni-limited-pod-density=true`) to enable nodes with more than 110 pods. 
```

Comment:
`to enable nodes with more than 110 pods` - this is only true if the node supports enough ENIs to allow more than 110 pods. For smaller instance sizes, you might still be capped at a number lower than 110.

Reply:
I added a sentence on this.

Reply:
I changed my mind. This concept should be intuitive to karpenter users. We are raising a limit.